### ISSUE TITLE
Pretrained weights for your version of Resnet101

### ISSUE THREAD

**Issue by lozino** [multimodallearning/pytorch-mask-rcnn#25]
Can the pretrained weights for your version of Resnet101 be found anywhere?

Thank you.
